src/Evolution/Systems/CurvedScalarWave/TimeDerivative.hpp

```diff
@@ -0,0 +1,92 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/FaceNormal.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Characteristics.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/Protocols.hpp"
```

Comment:
delete `Protocols.hpp`, `ProtocolHelpers.hpp`, `FaceNormal.hpp`,  `Options.hpp`, and `Magnitude.hpp`

Reply:
removed